icepyx/core/is2ref.py

```diff
@@ -12,6 +12,14 @@
 # ICESat-2 specific reference functions
 
 
+def _get_concept_id(product, version):
+    collections = earthaccess.search_datasets(
+        short_name=product, version=version, cloud_hosted=True
+    )
+    if collections:
+        return collections[0].concept_id()
+
+
```

Comment:
this function is duplicated (it also exists in class Query, line 139). Suggest removing this version.

Reply:
@betolink, reminder to remove the duplicated `_get_concept_id` function here in `is2ref.py`.